scripts/irods/test/test_resource_types.py

```diff
@@ -2109,6 +2109,36 @@ def test_get_stale_cache_replica__issue_6502(self):
             lib.remove_file_if_exists(stale_replica_filename)
             lib.remove_file_if_exists(get_filename)
 
+    def test_no_stage_to_cache_on_write__isue_6935(self):
+
+        try:
+
+            filename = 'testnostagetocacheonwrite.txt'
+            logical_path = os.path.join(self.admin.session_collection, filename)
+
+            # put the file to demoResc (compound)
+            self.admin.assert_icommand(['istream', 'write', '-R', 'demoResc', filename], input='testing issue 6935')
+
+            # repl the file to origResc (unixfilesystem)
+            self.admin.assert_icommand(['irepl', '-R', 'origResc', filename])
+
+            # trim the cache replica
+            self.admin.assert_icommand(['itrim', '-n0', filename], 'STDOUT', ['files trimmed = 1'])
+
+            # write a new file to origResc (unixfilesystem)
+            self.admin.assert_icommand(['istream', 'write', '-R', 'demoResc', filename], input='testing issue 6935 with more bytes')
```

Comment:
Is this supposed to be targeting `origResc`?

Reply:
No, it was not.  I also created a new resource in case you also just don't want origResc used.